locales-pending/premium.ftl

```diff
@@ -196,6 +197,19 @@ dashboard-exposures-filter-show-results = Show results
 dashboard-exposures-filter-reset = Reset
 
 
+dashboard-top-banner-scan-in-progress-title = Your scan is still in progress
+# Variables:
+# $data_breach_total_num is the total number of data breaches the user has.
+# $data_broker_total_num is the total number of data brokers selling the user’s data.
```

Comment:
Is this `$data_broker_total_num` variable used?

Reply:
Good catch — it’s not. Removed the comment in [f720dea](https://github.com/mozilla/blurts-server/pull/3431/commits/f720deae382fd2820ea384d686f8586bff84c59c).